gen/cycle.ml

```diff
@@ -720,8 +720,7 @@ let remove_store n0 =
       begin match m.evt.loc, m.evt.dir with
         | Code.Code _, Some R when not (E.is_ifetch m.edge.E.a1) ->
             if is_read_same_nonfetch m then begin
-              Printf.printf "%s" (str_node m);
```

Comment:
Would this be useful if it was gated by a debug flag?

Reply:
Hi @relokin thanks for comment! I'm not sure that a debug flag is necessary as using Warn.user_error will give the same result (as it will only be printed if debug mode is on, or if using diyone7, in which case it should indeed fail).

Reply:
ah I sorry I spoke too fast and didn't realise that you added the call to str_node in the user_error.